django_cloud_deploy/workflow/__init__.py

```diff
@@ -306,6 +311,18 @@ def update_project(self,
         if open_browser:
             webbrowser.open(app_url)
 
+    @staticmethod
+    def _sanitize_name(name: str) -> str:
```

Comment:
is this enough? What if the user uses non alphanumeric values? "#:)Break/Stuff "

Reply:
This function is only used to sanitize django_project_name. When prompting users for this, we use django_project_name.isidentifier() to validate it. See https://github.com/GoogleCloudPlatform/django-cloud-deploy/blob/master/django_cloud_deploy/cli/prompt.py#L192. This will make sure django_project_name only contains lowercase or uppercase characters, numbers or underscores.